docs/operator-manual/troubleshooting.md

```diff
@@ -0,0 +1,55 @@
+# Troubleshooting Tools
+
+The document describes how to use `argocd-tool` binary to simplify Argo CD settings customizations and troubleshot
+connectivity issues.
+
+## Settings
+
+Argo CD provides multiple ways to customize system behavior and has a lot of settings. It might be dangerous to modify
+settings on Argo CD used in production by multiple users. Before applying settings you can use `argocd-util` binary to
+make sure that settings are valid and Argo CD is working as expected. The `argocd-util` binary is available in `argocd`
+mage and might be used using docker. Example:
```

Comment:
should be `image` not `mage`. Further I think it might be a good idea to also tell that `argocd-util` can be used outside the image, by copying it from the docker image using `docker cp` or `kubectl cp`. Maybe we should make it available as asset from GitHub as well, since not everyone might have Docker installed on the workstation.

Reply:
done